apps/docs/content/tokens/semantic/color.mdx

```diff
@@ -12,10 +12,23 @@ export const tabs = [
   { title: "Dark", category: categoryKey }
 ];
 
+# Color
+
+## Guidelines
+
+Colours are one of the most obvious way to convey meaning in design. Depending on its context of use and on the amount of time we see them, we start to associate colour with meaning. Each of the colours used in the Workleap ecosystem complements each other and have a specific role to play.
```

Comment:
```suggestion
Colours are one of the most obvious ways to convey meaning in design. Depending on its context of use and on the amount of time we see them, we start to associate colour with meaning. Each of the colours used in the Workleap ecosystem complement each other and has a specific role to play.
```
Each is considered singular. I know, it's confusing sometimes. I had to look it up to remind myself.